src/main/java/com/zufar/onlinestore/cart/api/AddItemsToShoppingSessionHelper.java

```diff
@@ -72,7 +72,7 @@ private List<ShoppingSessionItem> createItems(Set<NewShoppingSessionItemDto> ite
                         .productsQuantity(productsWithQuantity.get(productInfo.getProductId()))
                         .productInfo(productInfo)
                         .build())
-                .collect(Collectors.toList());
+                .toList();
```

Comment:
be careful here. Should we return an immutable list here?

Reply:
Done